src/TTCache.php

```diff
@@ -81,14 +80,14 @@ public function remember(string $key, ?int $ttl, array $tags, callable $cb)
             $r = $this->tree->getFromCache($hkey);
             $this->tree->child($r->tags);
             $this->resetTree($isRoot);
-            return $r->value;
+            return Result::fromTaggedValue($r, true);
         }
 
         $r = $this->cache->get($hkey);
         if ($r) {
             $this->tree->child($r->tags);
             $this->resetTree($isRoot);
-            return $r->value;
+            return Result::fromTaggedValue($r, true);
```

Comment:
Is there no tag info of value we can include here?

Reply:
There is. `TaggedValue` is an internal class used for storing the data in MC. The `fromTaggedValue` factory method will grab the tags from that `TaggedValue` directly and apply them to the `Result`.